main.js

```diff
@@ -103,18 +138,100 @@ if (args.reqJs)
         console.log(edge.toString());
     });
 
-if (args.cg) {
-    function pp(v) {
-        if (v.type === 'CalleeVertex')
-            return astutil.ppPos(v.call);
-        if (v.type === 'FuncVertex')
-            return astutil.ppPos(v.func);
-        if (v.type === 'NativeVertex')
-            return v.name;
-        throw new Error("strange vertex: " + v);
+if (args.cgPath) {
```

Comment:
this is the main code to produce a call grapj json similar to what we already have for closure